lite/tests/segmentation/test_accuracy.py

```diff
@@ -0,0 +1,64 @@
+from valor_lite.segmentation import (
+    Accuracy,
+    DataLoader,
+    MetricType,
+    Segmentation,
+)
+
+
+def test_accuracy_basic_segmentations(basic_segmentations: list[Segmentation]):
+    loader = DataLoader()
+    loader.add_data(basic_segmentations)
+    evaluator = loader.finalize()
+
+    metrics = evaluator.evaluate(as_dict=True)
+
+    actual_metrics = [m for m in metrics[MetricType.Accuracy]]
+    expected_metrics = [
+        {
+            "type": "Accuracy",
+            "value": 2 / 3,
+            "parameters": {},
+        },
+    ]
+    for m in actual_metrics:
+        assert m in expected_metrics
+    for m in expected_metrics:
+        assert m in actual_metrics
+
+
+def test_accuracy_segmentations_from_boxes(
+    segmentations_from_boxes: list[Segmentation],
+):
+    loader = DataLoader()
+    loader.add_data(segmentations_from_boxes)
+    evaluator = loader.finalize()
+
+    metrics = evaluator.evaluate(as_dict=True)
+
+    actual_metrics = [m for m in metrics[MetricType.Accuracy]]
+    expected_metrics = [
+        {
+            "type": "Accuracy",
+            "value": 0.3334,
```

Comment:
can you help me understand the math behind .3334? I would have thought the accuracy would be higher given there's so many true negatives in these 270k pixel images.

Reply:
Updated the computation to account for background pixels being counted as True-Negatives (TN)